lib/Compiler.cpp

```diff
@@ -607,6 +607,9 @@ int PopulatePassManager(
     pm->add(clspv::createClusterPodKernelArgumentsPass());
   }
   pm->add(clspv::createReplaceOpenCLBuiltinPass());
+  if (clspv::Option::LongVectorSupport()) {
```

Comment:
If there are any phase-ordering dependencies you are aware of, please add a comment to indicate the restrictions here.

Reply:
Will do. Only ReplaceOpenCLBuiltinPass is required to my knowledge.